src/routes/home/Home.tsx

```diff
@@ -24,6 +28,7 @@ function Home() {
   const [openAbout, setOpenAbout] = useState(true);
   const [openAuthors, setOpenAuthors] = useState(true);
   const theme = useTheme();
+  const userApps = useAllStores() as StoreSummary[];
```

Comment:
Mysle ze mozesz to po prostu w hooku otypować jako StoreSummary[] bez null'a i wtedy tutaj juz nie trzeba